test/k8sT/Policies.go

```diff
@@ -1316,6 +1319,312 @@ var _ = SkipDescribeIf(func() bool {
 
 	})
 
+	Context("Basic Policy Metrics Test", func() {
```

Comment:
I appreciate the separation between test contexts done here, but I think the tradeoff of readability between test cases and added test runtime is not good here.

I think these new contexts should be merged into one since setup steps them seem to be exactly the same for each of them and just have 3 `It` blocks in the new context.